src/strings/normalize.h

```diff
@@ -75,9 +75,10 @@ MVM_STATIC_INLINE MVMint32 MVM_unicode_normalizer_process_codepoint(MVMThreadCon
      * far in normalized form without having to consider them into the
      * normalization process. The exception is if we're computing NFG, and
      * we got \r, which can form a grapheme in the case of \r\n. */
-    if (in < 0x20 || (in >= 0x7F && in <= 0x9F) || in == 0xAD)
-        if (!(MVM_NORMALIZE_GRAPHEME(n->form) && in == 0x0D))
+    if (in < 0x20 || (in >= 0x7F && in <= 0x9F) || in == 0xAD )
+        if (!(MVM_NORMALIZE_GRAPHEME(n->form) && in == 0x0D)) {
```

Comment:
Just started at this for a while trying to spot what changed... The answer seems to be "nothing worth changing".